packages/store/src/types.js

```diff
@@ -500,7 +500,10 @@
  * @property {() => Matcher} set A CopySet
  * @property {() => Matcher} bag A CopyBag
  * @property {() => Matcher} map A CopyMap
- * @property {() => Matcher} remotable A far object or its remote presence
+ * @property {(label?: string) => Matcher} remotable
+ * A far object or its remote presence. The optional `label` is purely for
+ * diagnostic purpose. It does not enforce any constraint beyond the
+ * must-be-a-remotable constraint.
```

Comment:
I think this is about the way jsdoc works, and not just WebStorm, but what I see is that anything left-justified is attributed to all the `@propoerties` above it. Text that is indented only applies to the one entry immediately above. 

```suggestion
 * @property {(label?: string) => Matcher} remotable - A far object or its remote presence.
 *   The optional `label` is purely for diagnostic purpose. It does not enforce any constraint
 *    beyond the must-be-a-remotable constraint.
```

@turadg am I right?

Reply:
Here's what I see hovering over this "remotable" in vscode.
<img width="954" alt="image" src="https://user-images.githubusercontent.com/273868/188286972-a1ec8de8-0afa-4f77-b5c2-d52b9235f999.png">

What do you see in WebStorm?

Reply:
Hi @Chris-Hibbert I'm still interested in investigating this anomaly. But for this PR, I'm resolving this conversation.

Reply:
![storeTypeHints](https://user-images.githubusercontent.com/13372636/188288965-1d5e2763-3850-4f7f-87f3-264e33232a92.png)

Reply:
Yuck!

Aside from the process question of how to do this in bulk, I don't see any reason not to do this. It adds value to the webstorm experience and does not subtract any value anywhere else. File an issue please, citing this conversation? Thanks.

Reply:
done